models/issue_reaction.go

```diff
@@ -247,3 +255,24 @@ func (list ReactionList) GetMoreUserCount() int {
 	}
 	return len(list) - setting.UI.ReactionMaxUserNum
 }
+
+// APIFormat returns Raction in api Format
+func (list ReactionList) APIFormat() api.CommentReactionList {
+	result := []*api.CommentReaction{}
+	users := make(map[string][]string)
+	counts := make(map[string]int64)
+
+	for _, r := range list {
+		users[r.Type] = append(users[r.Type], r.User.LoginName)
```

Comment:
r.User should be loaded before this invoked.

Reply:
I'll do `u := r.User` now
is this what you ment?